dev/NavigationView/NavigationView_InteractionTests/NavigationViewTests.cs

```diff
@@ -3630,6 +3630,69 @@ public void VerifyHeaderContentMarginOnTopNav()
             }
         }
 
+        [TestMethod]
+        [TestProperty("TestSuite", "D")]
+        public void VerifyHeaderContentMarginOnMinimalNav()
+        {
+            using (var setup = new TestSetupHelper(new[] { "NavigationView Tests", "NavigationView Test" }))
+            {
+                Log.Comment("Change display mode to left minimal");
+                var panelDisplayModeComboBox = new ComboBox(FindElement.ByName("PaneDisplayModeCombobox"));
+                panelDisplayModeComboBox.SelectItemByName("LeftMinimal");
+                Wait.ForIdle();
+
+                Log.Comment("Get HeaderContent Margin");
+                var getHeaderContentMarginButton = new Button(FindElement.ByName("GetHeaderContentMargin"));
+                getHeaderContentMarginButton.Invoke();
+                Wait.ForIdle();
+
+                var result = new TextBlock(FindElement.ByName("HeaderContentMarginResult"));
+                Verify.AreEqual(result.GetText(), "8,5,0,0");
```

Comment:
Can these tests be using the XML-based "visual verification" instead? That way if these numbers ever do change (intentionally) then they're easier to update.